src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Application.java

```diff
@@ -571,4 +571,9 @@ public Set<DocumentType> getMissingDocumentTypes() {
     public boolean isPaperCase() {
         return YES.equals(newPaperCase);
     }
+
+    @JsonIgnore
+    public String getPbaNumber() {
+        return this.getPbaNumbers().getValue().getLabel();
```

Comment:
does this return the selected pba number in drop down 🤔

Reply:
Yeah it does, this was initially in the PaymentService but since I needed the specific PBA for each payment I moved them to the relevant object - Application and FeeDetails objects

Reply:
Has all been tested locally as well